### ISSUE TITLE
Device locking time

### ISSUE THREAD

**Issue by dampfpunkt** [maelstrom2001/ArcticFox#393]
This is a great feature for safety. Prevents from auto fire in pocket when forgetting to turn device off. Would be nice if you could increase the 60sec limit.

**Comment by TBXin** [comment]
You can choose deep sleep behavior: lock device / turn off device. 

And the action will be performed after 3 minutes of inactivity.

**Comment by zalaross** [comment]
Yes indeed but it doesn't work on Eleaf pico if you have lsl on (in order to have a correct clock).

**Comment by TBXin** [comment]
Lsl does not affect device state.

**Comment by zalaross** [comment]
Then something else must prevent my pico from going to deep sleep (i have it lock the device).
